dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -158,6 +158,18 @@ public void testDashboard()
       // grpc-testing-1.17.1, shown as first item in linkage errors, has these errors
       Assert.assertTrue(linkage.getValue().contains("3 linkage errors in 3 classes"));
 
+      ImmutableList<Node> dependencyPaths = toList(
+          document.query("//p[@class='static-linkage-check-dependency-paths']"));
+      Node log4jDependencyPathMessage = dependencyPaths.get(dependencyPaths.size() - 1);
+      // Not to show all 900 dependency paths to log4j in the dashboard
+      Truth.assertThat(log4jDependencyPathMessage.getValue()).contains(
+          "and other 993 dependency paths to the jar file. All of them have the same artifacts:");
```

Comment:
I don't think we need "All of them have the same artifacts:"

Reply:
Simplified it.